analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentServiceFactory.java

```diff
@@ -2,12 +2,15 @@
 
 import java.util.Collections;
 
+import org.gradle.api.Project;
+
 public final class AlignmentServiceFactory {
 
 	private AlignmentServiceFactory() {
 	}
 
-	public static AlignmentService getAlignmentService() {
+	public static AlignmentService getAlignmentService(Project project) {
```

Comment:
Yes, this will be needed when we hook up to the dependency stuff (since the properties can have different effect based on which project they are used in)